content/articles/2018-08-05-google-summer-of-code-2018.md

```diff
@@ -0,0 +1,24 @@
+Title: GSoC'18: improving propagators, adding natural and artificial perturbations at Poliastro
+Date: 2018-08-05 14:00
+Category: GSoC
+Tags: GSoC, poliastro
+slug: 2018-08-05-google-summer-of-code-2018
+lang: en
+Author: Nikita Astrakhantsev
+
+It was a lazy sunday at home and nothing prepared me for my GF giving me the link for the GSoC'18 announcement page. I was considering participation a year ago, but had not enough resoluteness for that. This year I wanted it to be different so I started scrolling through the list of open-source organisations. As I am doing a PhD in theoretical physics now (defending in Jnauary 2019), I wanted the project to be scientific, not just some boring back-end development (in which I am anyway not strong enough). After a couple of iterations I converged to three space-aimed organisations. What shocked me, was the fact that mostly people of these organizations were not eager to even answer simple questions or it took them a week to provide a dataset for analysis. Only in Poliastro it was easy enough to get in touch with people and they (especially Juan) turned out to be friendly and helpful. In such an atmosphere it is more than possible to work.
+
+After looking project proposal list, I found what I am strong at and made a compilation of three project proposals into my own final [proposal](https://storage.googleapis.com/summerofcode-prod.appspot.com/gsoc/core_project/doc/5416846749073408_1521547584_proposal_astrakhantsev.pdf?Expires=1533564319&GoogleAccessId=summerofcode-prod%40appspot.gserviceaccount.com&Signature=OkUaCTO50Xy4dLVGKpUPlbsqj2XYI3QZRX8GriDapFoSpjV0xs%2F4zy2WD%2Fwj1dNZHxHYXppE7GR7i6D0cDjuarlWp6OwYPnoiBdCfh7Fg3UG8WjGWjOXE9oP8Jer2QA3teG5U9xYHgtStzTd9uQRA%2FA14qOQg0nVzJvCocIjYVWPROipz9V08uzhMbPISEX4AlGs2G5KC4F08P3D%2Bf4AaYsh6euug6JxY40zOrOZCwm%2Bg0Jh6s1kpB9XDeIzMb9Qdb06YFpLpQUggVNW7v7fPxuM188qEiNTdMR%2BvuMhnpgC1cV22%2FIB3l3xvPlFiJ%2BjpoDIw%2FX1JXHbfeQpl%2BUQzg%3D%3D) and got accepted.
```

Comment:
> `ExpiredTokenThe provided token has expired.Request has expired: 1533564319`

Not sure if there's a way to generate a public URL for the proposal? If not, it's a good moment to upload it to the wiki :)